README.md

```diff
@@ -1,8 +1,77 @@
+Apply engineering expertise to projects on devices and product development, programming and software design, and electronic manufacturing. 
+Collaborate on teams to manage multifaceted projects and synthesize complex concepts into engineering solutions. 
+Demonstrate proven ability to apply engineering skills to a wide range of electrical and software engineering projects. 
+
+## Technical Acumen
+
+**Proficient:** Java, Bash, SQL, Agile/Scrum, C,
+
+**Competent:** MATLAB, Python, Embedded System, C++, Junit, Lambda/Stream API, Mockito, Spring Framework
+
+**Familiar:** Google Cloud Platform, Git, Tensorflow
+
+**Hardware Design Tool:**  Cadence, ADS, HFSS, Simulink
+
+## Development Projects in Jarvis
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_shawn](https://github.com/jarviscanada/jarvis_data_eng_shawn)
+
+- **[Cluster Monitor](./linux_sql)**:   
+  * Developed a cluster monitoring bash agent that records hardware specifications and monitors
+resource usage (per minute) of all nodes in a cluster. Provisioned PostgreSQL database in nodes to store
+information of all nodes with the help of the bash agent installed in all of them. Utilized cron job to schedule the
+collection of resource usage information every minute. Coded SQL queries to generate reports for some business questions.
+                                      
+- **[Core Java Apps](./core_java)**:  
+  * Designed a Java8 based application called JavaGrep that searches all files in a directory
+recursively for a user-provided regular expression. Implemented method that contain those expressions and sent into a new file. 
+  * Developed a JDBC-based application that allows users to perform basic CRUD operations on a PostgreSQL database
+using DAO pattern. 
+  * Designed a Java8 based application called TwitterCLI which helps users create, read, and
+delete posts on Twitter using Twitter Rest API. Implemented four versions of the TwitterCLI application - basic Java
+version, Spring Beans version, Spring Component Scan version, and Springboot version. Wrote Unit Tests and
+Integration Tests for the applications using JUnit4 and Mockito.
+                                     
+- **[SpringBoot App](./springboot)**:  
+  * Implemented Java8 and Springboot based REST API of a trading platform that uses IEX Cloud
+as a data source and PostgreSQL to persist data. Used a three-tier microservice architecture to implement it. Made
+use of Spring DAO to handle data access in PostgreSQL. Wrote unit and integration test using JUnit4 and Mockito.
+Dockerized the application.   *in-progress*
+                                      
+- **[Cloud & DevOps](./cloud_devops)**: Not started
+- **[Hadoop](./hadoop)**: Not started
+- **[Spark/Scala](./spark)**:  Not started
+
+## Professional Experiences
+
+**Software Developer,  Jarvis, Toronto (2020-Present):**    
+Working on big data technologies based projects which include Java, Hadoop, Spark, Cloud, and DevOps. 
```

Comment:
Developing applications using the latest technologies, such as Java, Springboot, Cloud platforms, Hadoop, Spark, etc. 

write more for this job